kimchi/src/verifier.rs

```diff
@@ -762,28 +763,39 @@ pub fn verify<G, EFqSponge, EFrSponge>(
     group_map: &G::Map,
     verifier_index: &VerifierIndex<G>,
     proof: &ProverProof<G>,
+    public_input: &Vec<G::ScalarField>,
```

Comment:
I think all these `&Vec` should be `&[]` as it's more flexible (takes ref to array or vec)